src/deploy/get.rs

```diff
@@ -41,6 +42,35 @@ mod deploy_hash {
     }
 }
 
+/// Handles providing the arg for the retrieval of the finalized approvals.
+mod finalized_approvals {
+    use super::*;
+
+    const ARG_NAME: &str = "get-finalized-approvals";
+    const ARG_SHORT: char = 'a';
+    const ARG_VALUE_NAME: &str = "BOOLEAN";
+    const ARG_HELP: &str =
+        "An optional flag specifying whether the finalized approvals are retrieved";
+
+    pub(super) fn arg() -> Arg {
+        Arg::new(ARG_NAME)
+            .long(ARG_NAME)
+            .short(ARG_SHORT)
+            .required(false)
+            .value_name(ARG_VALUE_NAME)
```

Comment:
This has no effect as it's just a flag and takes no value.  It should be removed along with `ARG_VALUE_NAME`.